doc/make/index.md

```diff
@@ -54,7 +55,7 @@ below for details. Add `-jN` for an appropriate `N` to `make` for a parallel
 build.
```

Comment:
One more "below" above this line. There's also two adjacent mentions of `-j`.

Reply:
fixed